### DISCUSSION TITLE
Support Azure Cosmos NoSQL DB in Orleans version 7

### DISCUSSION THREAD

**Discussion by rothanzl** [dotnet/orleans#8412] [Q&A]
*Short question:*
Please can any one tell me why is Azure Cosmos NoSQL DB not supported in Orleans version 7?

*Longer question:*
Hello, we would like to migrate from orleans version 3 to version 7. We are currently storing data on Azure Cosmos NoSQL DB. 

The package *Orleans.Persistence.CosmosDB* is not supported in orleans v7. and on the other side package *Microsoft.Orleans.Persistence.AzureStorage* doesn't support Cosmos NoSql db.  

I didn't find a roadmap and I don't know whether is intended to not support NoSQL (and why), or is it planned to do it in the future, or any another rason exists.

**Comment by jurby** [comment]
Hi @galvesribeiro, is it covered by this PR - https://github.com/dotnet/orleans/issues/7726 ?
Is IT released in version 7.1, please? Thank you very much.

**Reply by ReubenBond** [reply]
It will be released in Orleans 7.2, likely next week
